### ISSUE TITLE
Package psbody-mesh missing file

### ISSUE THREAD

**Issue by Rajeswari-gn** [MPI-IS/mesh#72]
This package should have a AABB_tree.h file for installation, which seems to be missing (**this is the error I get during installation). Because of which, I am not able to install this package. How can this issue be sorted?

**Comment by lucasjinreal** [comment]
This lib seems unmaintained, please avoid using it.

**Comment by anuragranj** [comment]
Thanks @Rajeswari-gn . Whats the python version and os you are using? Also, what script did you run to install it? Can you paste the logs here?

**Comment by Rajeswari-gn** [comment]
Dear Anurag,

I am using Python 3.9 and Windows.

With Regards,
Raji

Rajarajeswari Ganesan
PhD. Student
Cardiovascular Biomechanics Group
Department of Biomedical Engineering
Eindhoven University of Technology


________________________________

**Comment by Rajeswari-gn** [comment]
> Thanks @Rajeswari-gn . Whats the python version and os you are using? Also, what script did you run to install it? Can you paste the logs here?

I am using Python 3.9 on Windows. I tried installing the package using setup.py file.  cd <path> and then pip install .

**Comment by Rajeswari-gn** [comment]
cd <path>
pip install .

**Comment by Rajeswari-gn** [comment]
cd "path to the folder"
pip install .

**Comment by anuragranj** [comment]
Firstly, this package is not supported on windows. Linux/macos are supported. You might need to check other issues to see if/how people have managed to run it on windows. Second, please look at the installation instructions, this package needs to be installed using `make all` and not pip.

**Comment by Rajeswari-gn** [comment]
I have also tried installation using "make all" in Ubuntu. But still I get this error.

**Comment by anuragranj** [comment]
Could you paste the full logs of your ubuntu installation?

**Comment by Rajeswari-gn** [comment]
$ python3 -m venv --copies my_venv
$ source my_venv/bin/activate
$ BOOST_INCLUDE_DIRS=/path/to/boost/include make all
The same way as described in the repository. But I get the error as "There is no make file."

**Comment by anuragranj** [comment]
This is still not full logs. Please attach full logs/errors.

**Comment by anuragranj** [comment]
Did you clone the repo? Are you in the root directory of the repo?

**Comment by Rajeswari-gn** [comment]
> This is still not full logs. Please attach full logs/errors.

Error Message: make: *** No targets specified and no makefile found.

**Comment by Rajeswari-gn** [comment]
> Did you clone the repo? Are you in the root directory of the repo?

Yes, I cloned the repository and specified the path to the root directory of the repo.

**Comment by anuragranj** [comment]
I am not able to reproduce this issue. It seems like the makefile is present here -
https://github.com/MPI-IS/mesh/blob/master/Makefile

**Comment by Rajeswari-gn** [comment]
> I am not able to reproduce this issue. It seems like the makefile is present here - https://github.com/MPI-IS/mesh/blob/master/Makefile

Yes, I checked the repo and the makefile is there in the folder, but still I get the error as mentioned above.

**Comment by raffienficiaud** [comment]
Can you show us the content of 

```
ls -al
```
from the location where you are running `make` ?

**Comment by Rajeswari-gn** [comment]
Thanks @anuragranj @raffienficiaud, the issue is resolved.

**Comment by Arish2022** [comment]
Hi @Rajeswari-gn , could you let me know how you solved this issue?

**Comment by Rajeswari-gn** [comment]
> Hi @Rajeswari-gn , could you let me know how you solved this issue?

Check the location of the makefile and root directory. Also check the path in the ubuntu environment while trying to build.
